tableauserverclient/models/interval_item.py

```diff
@@ -0,0 +1,99 @@
+import xml.etree.ElementTree as ET
+from datetime import datetime, time
+from .. import NAMESPACE
+
+
+class IntervalItem(object):
```

Comment:
I don't have an alternate suggestion, so feel free to ignore, but is IntervalItem the best name we've got?

The nested classes are fine for the enum-y purpose you're using it for (man, in py3 we could use real enums) -- but we might be hitting a point where we just needs a constants or common module that holds all these so the actual class with logic doesn't get muddled down.

I think this is also the case for RequestOptions?